examples/content/src/content/jcr_root/content/core-components-examples/en/.content.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I would name this path `/content/core-components-examples/library` if we don't plan to support multi-language yet.

Reply:
Good idea